lib/common/infrastructure/routes/providers.dart

```diff
@@ -15,7 +15,9 @@ import '../../domain/repositories/user/user_repository_facade.dart';
 
 /// Provides [StreamProvider] of app state changes
 final appStateStreamProvider = StreamProvider.autoDispose<AppState>((ref) {
-  var appState = const AppState();
+  var appState = const AppState(
+    isAuthenticated: true,
+  );
```

Comment:
Change the `appState` variable back to normal 

`var appState = const AppState();`